src/web/app/pages-admin/admin-home-page/admin-home-page.component.spec.ts

```diff
@@ -139,7 +143,7 @@ describe('AdminHomePageComponent', () => {
     ];
     fixture.detectChanges();
 
-    const button: any = fixture.debugElement.nativeElement.querySelector('#cancel-instructor-0');
+    const button: any = fixture.debugElement.nativeElement.querySelector('#remove-instructor-0');
```

Comment:
This button is no longer part of the home page component. There should be another way to trigger this event?

The same should apply to other buttons that are now part of the new component.

Reply:
Alright, updated.